crates/forge/bin/cmd/mod.rs

```diff
@@ -32,3 +32,4 @@ pub mod test;
 pub mod tree;
 pub mod update;
 pub mod watch;
+pub mod zksolc;
```

Comment:
nit: I think we always name the modules `zksync`... `zksolc` is for the compiler and `zk` for tests I believe (unofficially)